backends/rapidpro/task.go

```diff
@@ -32,15 +29,18 @@ func queueMsgHandling(rc redis.Conn, c *Contact, m *Msg) error {
 
 func queueChannelEvent(rc redis.Conn, c *Contact, e *ChannelEvent) error {
 	body := map[string]any{
-		"org_id":      e.OrgID_,
-		"contact_id":  e.ContactID_,
+		"event_id":    e.ID_,
+		"event_type":  e.EventType_,
```

Comment:
I want to switch these to all be a single `channel_event` task type, so step one is putting `event_type` in the payload